### ISSUE TITLE
Drush aliases and 'platform get' to a directory more than one level removed from the current location

### ISSUE THREAD

**Issue by vincenzo** [platformsh/platformsh-cli#364]
Right, I hope that the issue's title makes even sense :)

This is what my local deployment does amongst many things: 

```bash
$ platform get --no-build -e eID -p pID ~/path/to/destination
```

The result of this is that `platform` creates the drush files for the aliases under `~/.drush/home/user/path/to/destination.aliases.drushrc.php`. The CLI says that it created aliases in the group `@/home/user/path/to/destination` but `drush` just doesn't see those, whether I use `drush sa` or `platform drush-aliases`; or even just try and use the alias `@/home/user/path/to/destination` (which results in an error). 

Now, I am not a full-fledged guru of `drush` but I am keen to believe that those aliases will never be picked up if stored in subdirectories of `~/.drush`, hence the rule for generating aliases files should change. In which case, I have an idea and I can come up with a PR pretty quickly, I think. 

Thoughts, @pjcdawkins?

**Comment by pjcdawkins** [comment]
Hi @vincenzo, you're right that was totally wrong, I've made a PR (#365) that should address this

**Comment by vincenzo** [comment]
Hi @pjcdawkins - thanks for that. 
That looks good to me and it should indeed fix the issue. 
However, you may be left with a latent issue (that could potentially never occur, since you generate the aliases on 'get'). Nonetheless, if I `cd` to `~/path/to/destination` (after I `platform get` it) and there are no aliases for it (in my setup, all the sites I fetched so far have no aliases, since the ones created are undetectable), when I run `platform drush-aliases`, it will default in 'create' mode and create the aliases. Guess where? In `~/.drush/home/user/path/to/destination.aliases.drushrc.php`. All this to say that you may want to look at how `drush-aliases` works out the path of the file to store the aliases in. Still, I'd also leave in the fix from your PR #365, as it guarantees the shortest possible name for the alias group, I believe. That is, unless you choose to move that fix on the `drush-aliases` level.

**Comment by pjcdawkins** [comment]
I think that's OK, the default alias group for `drush-aliases` is:

1. whatever is specified in the `-g` or `--group` option
2. or whatever is defined in the `.platform-project` file
3. or the project ID

At the moment you're getting the full path from (2), and you can change it with `-g`

**Comment by vincenzo** [comment]
:+1:

**Comment by pjcdawkins** [comment]
Fixed in v2.6.0
